fudy/NoteScreen/Component/FramelessWindow.qml

```diff
@@ -111,15 +114,50 @@ ApplicationWindow {
 		onDoubleClicked: function() {
 			root.doubleClicked();
 		}
+
+		onPositionChanged: function(mouse) {
+			if (mouse.x >= parent.width - internal.margin && mouse.y >= parent.height - internal.margin) {
```

Comment:
It is long line and makes us confused.

Reply:
![Screenshot 2024-07-24 005154](https://github.com/user-attachments/assets/63202e21-9859-4650-9f1c-660d278ebf88)
The red area is the zone that allows the mouse to change its shape.